lms/static/scripts/frontend_apps/utils/google-api-client.js

```diff
@@ -1,3 +1,28 @@
+/**
+ * Load the Google Identity Services client library, if not already loaded.
+ *
+ * See https://developers.google.com/identity/oauth2/web/guides/load-3p-authorization-library.
+ *
+ * @return {Promise<typeof window.google.accounts>}
+ */
+export async function loadIdentityServicesLibrary() {
+  if (window.google?.accounts) {
+    return window.google.accounts;
+  }
+
+  return new Promise((resolve, reject) => {
+    const script = document.createElement('script');
+    script.src = 'https://accounts.google.com/gsi/client';
+    script.onload = () => {
+      resolve(window.google.accounts);
```

Comment:
We might want to be more explicit in a comment somewhere that this script (well, I presume) sets `window.google.accounts`.

Reply:
I'd like to treat that as an implementation detail of this function, so that code outside the module doesn't touch this global, except for the tests.